core/include/kernel_macros.h

```diff
@@ -16,6 +16,9 @@
  * @author      René Kijewski
  */
 
+#ifndef KERNEL_MACROS_
```

Comment:
the diff of this file could be completely moved to its own PR. seems unrelated to OLSR to me.